templates/konvoyconfig.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoyconfig
+  namespace: kubeaddons
+  labels:
+    kubaddons.mesosphere.io/name: konvoyconfig
+  annotations:
+    appversion.kubeaddons.mesosphere.io/konvoyconfig: "0.0.1"
+    values.chart.helm.kubeaddons.mesosphere.io/konvoyconfig: "https://raw.githubusercontent.com/mesosphere/charts/360fcc627d014aca967404891fcd55da75ba0820/values.yaml"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: konvoyconfig
+    repo: https://mesosphere.github.io/charts/staging
+    version: 0.0.2
+    values: |
+      ---
+      clusterHostname: ""
```

Comment:
:thinking: would we want to add a small comment here explaining to the reader the full context of how and why this configuration option works like this?